texthero/preprocessing.py

```diff
@@ -24,32 +24,39 @@
 
 
 @InputSeries(TextSeries)
-def fillna(s: TextSeries) -> TextSeries:
+def fillna(s: TextSeries, replace_string: Optional[str] = "") -> TextSeries:
```

Comment:
I wouldn't do the optional type here, either the "" or the typing with Optional[str] but not both, as we don't gain any informations from that.